app/build.gradle

```diff
@@ -43,4 +46,8 @@ dependencies {
     compile 'com.squareup.retrofit2:converter-gson:2.0.0'
     compile 'com.squareup.okhttp:logging-interceptor:2.6.0'
     testCompile 'junit:junit:4.12'
+    testCompile 'org.mockito:mockito-core:1.10.8' //Versions above are not supported by powermock
```

Comment:
Actually... This may be incorrect. Let's hold off on the merge until I get home (a couple of hours).